build/linux/rpmDependenciesGenerator.js

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+const { spawnSync } = require('child_process');
+const { calculatePackageDeps, mergePackageDeps } = require('./linux-installer/rpm/rpmDependencyScripts');
+const { resolve } = require('path');
+const { readFileSync } = require('fs');
+
+// @ts-check
+function getRpmDependencies() {
+	// Get the files for which we want to find dependencies.
+	const findResult = spawnSync('find', ['.', '-name', '*.node']);
```

Comment:
Can you run this step on build generated under the `agent.buildDirectory` similar to https://github.com/microsoft/vscode/blob/32d40cf44e893e87ac33ac4f08de1e5f7fe077fc/build/darwin/sign.ts#L29